src/PointwiseFunctions/AnalyticSolutions/RadiationTransport/M1Grey/ConstantM1.cpp

```diff
@@ -140,6 +170,34 @@ GENERATE_INSTANTIATIONS(INSTANTIATE_M1_FUNCTION,
 #undef EBIN
 #undef GENERATE_LIST
 
+#define TAG(data) BOOST_PP_TUPLE_ELEM(0, data)
+#define NTYPE(data) BOOST_PP_TUPLE_ELEM(1, data)
+#define EBIN(data) BOOST_PP_TUPLE_ELEM(2, data)
+#define GENERATE_LIST(z, n, _) BOOST_PP_COMMA_IF(n) n
+
+#define INSTANTIATE_M1_FUNCTION(_, data)                                       \
+  template tuples::TaggedTuple<TAG(data) < NTYPE(data) < EBIN(data)>>>         \
+      ConstantM1::variables(const tnsr::I<DataVector, 3>& x, double t,         \
+                            tmpl::list<TAG(data) < NTYPE(data) < EBIN(data)>>> \
+                            /*meta*/) const noexcept;
+
+#define temp_list \
```

Comment:
This is defined twice and never undefined.  Go through the macros here and make sure you undefine them all.